addons/website_customer/static/tests/tours/customer_filter_with_tag.js

```diff
@@ -0,0 +1,58 @@
+import { clickOnElement, registerWebsitePreviewTour } from "@website/js/tours/tour_utils";
+import { queryAll } from "@odoo/hoot-dom";
+
+registerWebsitePreviewTour(
+    "customer_filter_with_tag_tour",
+    {
+        url: "/customers",
+    },
+    () => [
+        clickOnElement("tag A", ":iframe .card-body .o_customer_tag:contains('Tag A')"),
+        {
+            trigger: ":iframe .customer_row",
+            run() {
+                const cards = queryAll(".customer_card", { root: this.anchor });
+                if (!cards.length) {
```

Comment:
Since only one customer has "Tag A" in the test data, we can assert that directly:
```suggestion
                if (cards.length !== 1) {
```